assets/resourceSnippets/Automanage Best Practices DevTest Assignment.snippet.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please make the following change to test/functional/snippets.test.ts in order to turn off schema validation during tests for these snippets:

```ts
// Override whether to ignore schema validation - default is to perform schema validation - mark with "true" to ignore schema validation
// TODO: All items in this list indicate an error (either snippet or schema) and should eventually be removed
const overrideIgnoreSchemaValidation: { [name: string]: boolean } = {
    /* // TODO: fix snippet
     +   'Missing required property "protectedSettings"'
     */
    "Linux VM Custom Script": true,

==>>> ADD THESE LINES:
    // TODO: Remove when schemas are published
    "Automanage Best Practices DevTest Assignment": true,
    "Automanage Best Practices Prod Assignment": true,
    "Automanage Create Custom Profile": true,
    "Automanage Custom Profile Assignment": true,
};
```

Reply:
Done